collab-folder/src/core/view.rs

```diff
@@ -478,8 +489,21 @@ pub struct View {
   #[serde(default)]
   pub is_favorite: bool,
   pub layout: ViewLayout,
-  pub icon_url: Option<String>,
-  pub cover_url: Option<String>,
+  pub icon: Option<ViewIcon>,
+}
+
+#[derive(Eq, PartialEq, Debug, Hash, Clone, Serialize_repr, Deserialize_repr)]
+#[repr(u8)]
+pub enum IconType {
```

Comment:
It would be better if you specify the value of the enum.
<img width="693" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy-Collab/assets/86001920/a27effce-cf0e-43f1-a6b2-6abbf883067f">